content/source/docs/enterprise/api/organizations.html.md

```diff
@@ -221,6 +221,8 @@ curl \
 
 `PATCH /organizations/:organization_name`
 
+-> **Note**: Renaming an organization is not currently supported if that organization also relies on the Private Module Registry. Existing modules will need to be deleted and later reloaded to accomplish this action. This functionality will be fully supported in a future release.
```

Comment:
I think Ryan's suggestion of "isn't recommended if the organization has modules in the Private Module Registry" is the way I would go here. I'd also be more explicit in the second sentence:

"To rename an organization with private modules, first, delete the modules from the Private Module Registry. Then make the name change. After the change is complete, reload the modules into the Private Module Registry."

And I wouldn't say anything about the future. Customers email us asking "when will that happen" if you do. :) We can just delete the note when it happens.

Reply:
Updated